### PR TITLE
added 03_run_test.yml

### LINKED ISSUE #3: Add tests

### LINKED ISSUE #22: Fix - ACL problems consul_acl_default_policy=deny
https://github.com/fredrikhgrelland/vagrant-hashistack/issues/344

### PR DESCRIPTION
**Pull request by dangernil**
closes #3 
closes #22 

Added 03_run_test.yml

### CODE REVIEW

**Comment by Neha-Sinha2305** [comment]
@dangernil : You need to send in the consul_token when making the healthcheck call when consul_acl_default_policy is set to **deny**.
Can refer to the healthcheck code in minio repo: https://github.com/fredrikhgrelland/terraform-nomad-minio/blob/master/dev/ansible/03_run_tests.yml
And info on why this is required now can be found in this issue: https://github.com/fredrikhgrelland/terraform-nomad-minio/issues/26

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 ### Added
 
 - Using Vault to generate username and password for postgres with kv #13
+- Added 03_run_tests.yml #3
+- Code to support successful execution of nomad mc job and tests when consul_acl_default_policy is deny
```
Comment (zhenik) [inline CHANGELOG.md]:
```suggestion
- Code to support successful execution of nomad mc job and tests when consul_acl_default_policy is deny #22
```

File: Makefile [hunk 2]
```diff
@@ -46,6 +46,9 @@ remove-tmp:
 
 clean: destroy-box remove-tmp
 
+proxy:
+	consul intention create -token=master postgres-local postgres
```
Comment (zhenik) [inline Makefile:85]: Keep in mind that this command is not idempotent. Could you test it? (Run 2-3 times)

**Review by zhenik — CHANGES_REQUESTED** [review]

**Review by zhenik — APPROVED** [review]

**Review by claesgill — APPROVED** [review]
LGTM 🚀

**Pull request merged**